### PR TITLE
Irish Language

### PR DESCRIPTION
**Pull request by BhargavaReddy24**
need some more development for this python code and changes 
this is good for now

### CODE REVIEW

File: irishlang.py [hunk 1]
```diff
@@ -0,0 +1,63 @@
+import math
+import ast
+class irishspell:
+    def __init__(self):
+        self.file1 = open("irish.txt", "r", encoding = "UTF-16")
```
Comment (maddiekoehler) [inline irishlang.py:5]: Will you upload this irish.txt file as well so I can run this?

File: irishlang.py [hunk 2]
```diff
@@ -0,0 +1,63 @@
+import math
+import ast
+class irishspell:
+    def __init__(self):
+        self.file1 = open("irish.txt", "r", encoding = "UTF-16")
+        self.corpus1 = self.file1.read()
+        self.frequency_dictionary = ast.literal_eval(self.corpus1)
+        self.file1.close()
+        self.file3 = open("frequency.txt", "r", encoding = "UTF-16")
```
Comment (maddiekoehler) [inline irishlang.py:9]: Will you upload this file too please?

**Review by maddiekoehler — APPROVED** [review]

File: irishlang.py (hunk 1 above)
Comment (BhargavaReddy24) [reply irishlang.py:5]: [irish.txt](https://github.com/maddiekoehler/spellchecker-team5/files/7556842/irish.txt)

File: irishlang.py (hunk 2 above)
Comment (BhargavaReddy24) [reply irishlang.py:9]: [frequency.txt](https://github.com/maddiekoehler/spellchecker-team5/files/7556849/frequency.txt)

**Pull request merged**